### PR TITLE
Python 3.13 initial support

### PR DESCRIPTION
**Pull request by greenozon**
This is an early Python 3.13 initial support
I've been playing around starting from alpha builds, there were lots of changes each alpha version, 
but starting from beta versions the python-guys announced a promise:

```
3.13.0 beta 1: Wednesday, 2024-05-08 (No new features beyond this point.)
```
it's from here:  https://peps.python.org/pep-0719/

each beta versions (beta 1/2/3) did not change any opcodes (no new added, no old removed, etc) which is good

thus, I've collected new opcodes and added support for early 3.13
to be honest, the range of new opcodes is very impressive :)

pycdas: able to print opcodes disassembly as usually
pycdc: need to gradually add support for new opcodes (and align changed logic for existing ones,.

PS maybe there will be a need to update MAGIC_3_13  value (but it did not change between b2 -> b3)

### CODE REVIEW

File: bytes/python_3_13.cpp [hunk 1]
```diff
@@ -0,0 +1,216 @@
...
+    MAP_OP(17, RESERVED)
+    MAP_OP(18, GET_ANEXT)
+    MAP_OP(19, GET_ITER)
+    MAP_OP(20, GET_LEN)
+    MAP_OP(21, GET_YIELD_FROM_ITER)
+    MAP_OP(22, INTERPRETER_EXIT)
+    MAP_OP(23, LOAD_ASSERTION_ERROR)
+    MAP_OP(24, LOAD_BUILD_CLASS)
+    MAP_OP(25, LOAD_LOCALS)
+    MAP_OP(26, MAKE_FUNCTION_A) //??
```
Comment (zrax) [inline bytes/python_3_13.cpp:30]: Python 3.13 no longer uses oparg for MAKE_FUNCTION, so this should be a new mapped opcode

```suggestion
    MAP_OP(26, MAKE_FUNCTION)
```

File: bytecode_ops.inl [hunk 2]
```diff
@@ -118,6 +118,14 @@ OPCODE(RESERVED)                        // Python 3.12 ->
 OPCODE(BINARY_SLICE)                    // Python 3.12 ->
 OPCODE(STORE_SLICE)                     // Python 3.12 ->
 OPCODE(CLEANUP_THROW)                   // Python 3.12 ->
+OPCODE(BINARY_OP_INPLACE_ADD_UNICODE)   // Python 3.13 ->
+OPCODE(EXIT_INIT_CHECK)                 // Python 3.13 ->
+OPCODE(FORMAT_SIMPLE)                   // Python 3.13 ->
+OPCODE(FORMAT_WITH_SPEC)                // Python 3.13 ->
+OPCODE(TO_BOOL)                         // Python 3.13 ->
```
Comment (zrax) [inline bytecode_ops.inl:125]:
```suggestion
OPCODE(FORMAT_WITH_SPEC)                // Python 3.13 ->
OPCODE(MAKE_FUNCTION)                   // Python 3.13 ->
OPCODE(TO_BOOL)                         // Python 3.13 ->
```

File: bytecode_ops.inl [hunk 3]
```diff
@@ -118,6 +118,14 @@ OPCODE(RESERVED)                        // Python 3.12 ->
...
+OPCODE(UNPACK_SEQUENCE_LIST)            // Python 3.13 ->
+OPCODE(UNPACK_SEQUENCE_TUPLE)           // Python 3.13 ->
+OPCODE(UNPACK_SEQUENCE_TWO_TUPLE)       // Python 3.13 ->
```
Comment (zrax) [inline bytecode_ops.inl:128]: These three are below the HAVE_ARG marker (now moved to 44), and actually use their oparg, so they should be `_A` variants

File: bytecode_ops.inl [hunk 4]
```diff
@@ -272,3 +360,4 @@ OPCODE_A(INSTRUMENTED_END_FOR)              // Python 3.12 ->           (see END
 OPCODE_A(INSTRUMENTED_END_SEND)             // Python 3.12 ->           (see END_SEND)
 OPCODE_A(INSTRUMENTED_INSTRUCTION)          // Python 3.12 ->           ???
 OPCODE_A(INSTRUMENTED_LINE)                 // Python 3.12 ->           ???
+OPCODE_A(INSTRUMENTED_CALL_KW)              // Python 3.13 ->           ???
```
Comment (zrax) [inline bytecode_ops.inl:363]: Newline at EOF

File: .gitignore [hunk 5]
```diff
@@ -6,3 +6,15 @@
...
+CMakeCache.txt
+cmake_install.cmake
+*.user
+*.filters
+*.vcxproj
+*.sln
+.vs/
+Debug/
+x64/
+*.dir/
```
Comment (zrax) [inline .gitignore:20]: If you want to add to the default gitignore, that should be in a separate PR.
```suggestion
```

**Comment by greenozon** [comment]
@zrax   thanks for review!

**Review by zrax — APPROVED** [review]
